.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs and Storybook
+on:
+    push:
+        branches:
+            - master
+jobs:
+    publish:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  fetch-depth: 1
+            - name: Configuring Node.js
+              uses: actions/setup-node@v2-beta
+              with:
+                  node-version: '10.15.3'
+            - name: Installing dependencies
+              run: |
+                  npm install
+                  npm install -g nx
+            - name: Run NX Commands
+              run: |
+                  nx run dotcms-webcomponents:build
```

Comment:
Why do we need this? are we publishing `dotcms-webcomponents` docs too? If so... NICE!

Reply:
Because the Storybook config of dotcms-webcomponents requires `defineCustomElements` from `loader` which is output to dist.